File: DTOs/Sources/AuthServiceDTOs/UserProfileUpdateDTO.swift
import Foundation

public struct UserProfileUpdateDTO: Codable {
    public var name: String?
    public var profileImage: Data?
    public var isNotificationsActive: Bool?
    public var isPushNotificationsActive: Bool?
    
    public init(name: String? = nil, profileImage: Data? = nil, isNotificationsActive: Bool? = nil, isPushNotificationsActive: Bool? = nil) {
        self.name = name
        self.profileImage = profileImage
        self.isNotificationsActive = isNotificationsActive
        self.isPushNotificationsActive = isPushNotificationsActive
    }
}

File: android/app/src/main/java/net/ipv64/kivop/dtos/AuthServiceDTOs/UserProfileUpdateDTO.kt
package net.ipv64.kivop.dtos.AuthServiceDTOs

import java.util.UUID
import java.time.LocalDateTime

data class UserProfileUpdateDTO (
    var name : String?,
    var profileImage : String?,
    var isNotificationsActive : Boolean?,
    var isPushNotificationsActive : Boolean?,
)
